nyx/chain.json

```diff
@@ -0,0 +1,186 @@
+    "persistent_peers": [
+      {
+        "id": "4acf579e2744268f834c713e894850995bbf0ffa",
+        "address": "50.18.31.225:26656"
+      }, 
+      { 
+        "id": "33f8e220cc44f2fed6a839b55f4520f0cf6fa8b2",
+        "address": "142.132.147.189:38656"
+      }, 
+      {
+        "id": "a58b9b102e7d91f4900036b40b8232d386245c55", 
+        "address": "3.65.247.65:26656"
+      }, 
+      {
+        "id": "5185b9f7abedbbbf48149491731fe9b6aa39b17b",
+        "address": "38.242.194.49:26656"
+      },
+      {
+        "id": "ee03a6777fb76a2efd0106c3769daaa064a3fcb5",
+        "address": "51.79.21.187:26656"
+      },
+      {
+        "id": "6bac8d8e4b43a230d2da45480d82ec87144c2f23",
+        "address": "nyx-validator.nym.blockfend.com:26656"
+      },
+      {
+        "id":"3905ef90a12784a6f0ae4435fcd8ad13816f12ac", 
+        "address": "35.245.211.169:26656"
+      },
+      {
+        "id": "510cdd3e5ed0367eafa3d1c277ede2e0a4536dbb", 
+        "address": "35.215.58.39:26656"
+      },
+      {
+        "id": "ded5157860dcd05e6308ef453df9bd53c1112bb6",
+        "address": "35.215.44.229:26656"
+      },
+      {
+        "id": "2f57a4a5926ceadb91a6df09a82ebad322777a6c", 
+        "address": "sentry-2.nym.ezstaking.xyz:26666"
+      },
+      {
+        "id": "8fb63514b6325459fb90ede3b3697ec0827b6f50", 
+        "address": "35.223.70.235:26656"
+      }, 
+      {
+        "id": "6625c048105d86d8bd6c22bc1359e10f4de1e6d4", 
+        "address": "34.125.78.165:26656"
+      }, 
+      {
+        "id": "7c5a3701cb4095fd28484e39d29e270001c19ae1",  
+        "address": "65.108.110.59:38656"
+      },
+      {
+        "id": "a0fd38e40fd53699437d90456e0a2e62df0607e0", 
+        "address": "15.235.9.199:26656"
+      }, 
+      {
+        "id": "e6d01a3ab3467953d14ad79a56a5a5eec0625096",
+        "address": "35.215.56.11:26656"
+      },
+      {
+        "id": "22120fe1203d2721cda9dd2548a346fdae43aedb", 
+        "address": "51.79.80.25:26656"
+      }, 
+      {
+        "id": "611c8b24adc5ae17477b3dfedae32af117d6d276", 
+        "address": "35.215.45.32:26656"
+      }, 
+      {
+        "id": "4e1663adac48c439baa5fbaebb9c9ddde91f9c18", 
+        "address": "51.79.72.10:26656"
+      }, 
+      {
+        "id": "b436cf49cd0e163ea1aa9eaf38e3d2250eb42c9e", 
+        "address": "51.222.248.217:26656"
+      },
+      {
+        "id": "2e2ae7a117b22c0a09b685b963603150f702b6af", 
+        "address": "144.126.133.37:10053"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address":"https://rpc.nyx.nodes.guru/",
+        "provider":"nodes guru"
+      }, 
+      {
+        "address":"https://nym.vladiatorlabs.io/", 
+        "provider":"vladiatorlabs"
+      }, 
+      {
+        "address":"https://rpc.nyx.ezstaking.io/", 
+        "provider":"ezstaking"
+      }, 
+      {
+        "address":"https://rpc.nym.forbole.com/", 
+        "provider":"forbole"
+      }, 
+      {
+        "address":"https://rpc-nyx.testnet.run/", 
```

Comment:
is this one just for a testnet? we have a testnets folder for that, but shouldn't be included here if it's not for mainnet

Reply:
No, its a mainnet validator called 'testnet run', which i understand is a little confusing. Its for mainnet though.